source/reference/limitations.txt

```diff
@@ -7,24 +7,47 @@ Limitations
 .. include:: /includes/preview-warning
 
 {+c2c-full-product-name+} is under active development. This preview
-release has several limitations.
+release has several limitations. Review the list for changes after the
+{+c2c-product-name+} General Availability release.
+
+``mongosync`` does not check for compliance with the following
+limitations. Please ensure that your application is not affected by
+these limitations. Running ``mongosync`` in the presence of one of
+these limitations could lead to undefined behavior on the destination
+cluster.
 
 General Limitations
 -------------------
 
-- The minimum supported version is MongoDB 6.0.
-- The minimum support Feature Compatibility Version is 6.0. 
-- MongoDB Community Edition is not supported.
+- The minimum supported server version is MongoDB 6.0.
 - The source and destination clusters must have the same release
   version.
-- Environmental assumptions are not validated as pre-checks. 
-- The Primary cluster cannot be changed while syncing.
-- If write blocking is disabled,the client must prevent writes to the
-  source cluster after starting the commit process.
+- The minimum supported Feature Compatibility Version (fCV) is 6.0.
```

Comment:
I think we should link to the FCV docs like https://www.mongodb.com/docs/v6.0/reference/command/setFeatureCompatibilityVersion/ here. FCV isn't so well known of a concept.

Reply:
fCV or FCV? I've seen it a few different ways across the docs. 

Ok deleting the acronym as well

Reply:
Done